server/docs/authors-haven-api.yml

```diff
@@ -50,6 +50,53 @@ paths:
           description: Conflict. Email or username exists
         500:
           description: Internal server error
+  /api/v1/sessions/create:
+    post:
+      summary: Login Route
+      description: Allow existing users to login to use the application
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              required:
+              - email
+              - password
+              properties:
+                userLogin:
+                  type: string
+                  minLength: 2
+                  example: JhayX
+                password:
+                  type: string
+                  minLength: 4
+                  example: incorrect
```

Comment:
Ade, I am aware that there were no validation rules at the time you wrote this documentation. However, there is an adopted rule now. Could you please reflect them here in your documentation?